### ISSUE TITLE
export mp4 is broken on 0.3.2

### ISSUE THREAD

**Issue by ParvinEyvazov** [CapSoftware/Cap#204]
Hey, it is an awesome project! Best Loom competitor out there. Features are great and upcoming features are awesome!

I have an issue with the latest version.

### Description

Export mp4 is broken. Tried with 2 macs (silicon). For both, couldn't open the file in anywhere.

### Reproduction

Nothing exceptional, started the recording and stopped it. Did export, and the file is broken.

### Additional Context

- Cap version: 0.3.2
- Operating system, version: Mac (Silicon)
- Device (optional): M1 pro | M1 air

<img width="752" alt="Screenshot 2024-12-08 at 23 17 01" src="https://github.com/user-attachments/assets/d9a817df-fe10-4051-9973-73a71d207626">

**Comment by Brendonovich** [comment]
If you go to `/Users/{your user}/Library/Application Support/so.cap.desktop/recordings` and open the recording using 'Show Package Contents', is there a `result/output.mp4` file? We suspect that exporting itself isn't broken, but rather the copying from the recording bundle to the target destination.

<img width="471" alt="image" src="https://github.com/user-attachments/assets/0b7d90df-5f2c-4982-b824-03a92250e33e">

<img width="486" alt="image" src="https://github.com/user-attachments/assets/250a31f5-2121-4280-9d47-c7adc3538275">

**Comment by ParvinEyvazov** [comment]
Exactly! The file `result/output.mp4` works fine, but the one copied to the desktop is not.

**Comment by ifielder** [comment]
Same issue

**Comment by Brendonovich** [comment]
This should be fixed in >0.3.3
